google-ads/src/main/java/com/google/ads/googleads/lib/GoogleAdsClient.java

```diff
@@ -149,16 +168,97 @@ public static Builder newBuilder() {
      */
     public abstract Builder setCredentials(Credentials credentials);
 
-    private void setCredentials(Properties properties) {
-      UserCredentials credentials =
-          UserCredentials.newBuilder()
-              .setClientId(properties.getProperty(ConfigPropertyKey.CLIENT_ID.getPropertyKey()))
-              .setClientSecret(
-                  properties.getProperty(ConfigPropertyKey.CLIENT_SECRET.getPropertyKey()))
-              .setRefreshToken(
-                  properties.getProperty(ConfigPropertyKey.REFRESH_TOKEN.getPropertyKey()))
-              .build();
-      setCredentials(credentials);
+    private void setCredentials(Properties properties) throws IOException {
```

Comment:
Nit: Is it worth providing a setCredentials(Credentials creds)? That would allow users to load their own creds that say, they share with another Cloud API?